benchmark/primitives/primitives_test.go

```diff
@@ -188,6 +189,59 @@ func BenchmarkMutexWithoutDefer(b *testing.B) {
 	}
 }
 
+func BenchmarkIncrInt64Atomics(b *testing.B) {
```

Comment:
How about `BenchmarkAtomicAddInt64` to line up with the existing benchmark names?

Reply:
done.